src/main/java/db/migration/V166__Ensure_Activities_Of_Numeric_SectionGroups_Are_Associated_To_SectionGroup.java

```diff
@@ -0,0 +1,114 @@
+package db.migration;
+
+import edu.ucdavis.dss.ipa.api.helpers.Utilities;
+import org.flywaydb.core.api.migration.jdbc.JdbcMigration;
+
+import java.sql.*;
+
+public class V166__Ensure_Activities_Of_Numeric_SectionGroups_Are_Associated_To_SectionGroup implements JdbcMigration {
+
+    @Override
+    public void migrate(Connection connection) throws Exception {
+        // Find all activities
+        // Is the activity associated to a section?
+        // If the section has a numeric sequenceNumber, add a sectionGroup association and remove the section association
+
+        PreparedStatement psActivities = connection.prepareStatement("SELECT * FROM `Activities`");
+
+        try {
+            connection.setAutoCommit(false);
+
+            ResultSet rsActivities = psActivities.executeQuery();
+
+            while (rsActivities.next()) {
+                long activityId = rsActivities.getLong("Id");
+                long sectionId = rsActivities.getLong("SectionId");
+                long sectionGroupId = rsActivities.getLong("SectionGroupId");
+
+                if (sectionGroupId > 0) {
```

Comment:
Does this imply we're skipping shared activities? If so, aren't false shared activities what we're trying to clean up?

Reply:
Our current architecture is that a numeric sectionGroup (which can only have one section) should have its activities tied to the sectionGroup.

A letter based sectionGroup would have activities on the section for section specific activities, and shared/universal activities would go on the sectionGroup.

I think the original idea was that activities universal to all sections should go on the sectionGroup in both cases.